core/src/avm2/class.rs

```diff
@@ -6,9 +6,29 @@ use crate::avm2::r#trait::{Trait, TraitKind};
 use crate::avm2::script::TranslationUnit;
 use crate::avm2::string::AvmString;
 use crate::avm2::{Avm2, Error};
+use enumset::{EnumSet, EnumSetType};
 use gc_arena::{Collect, GcCell, MutationContext};
 use swf::avm2::types::{Class as AbcClass, Instance as AbcInstance};
 
+#[derive(Clone, Debug, Collect)]
+#[collect(require_static)]
+pub struct CollectWrapper<T>(T);
```

Comment:
I feel like I need to find a way to conditionally derive `Collect` so that `swf` can expose it as a crate feature item and `core` can request garbage-collectable SWF types... but that's a PR for another time.